pkg/build/gobuild.go

```diff
@@ -872,9 +873,12 @@ func (g *gobuild) buildAll(ctx context.Context, ref string, baseIndex v1.ImageIn
 		return nil, err
 	}
 
+	errg, ctx := errgroup.WithContext(ctx)
```

Comment:
I think we want this to share it's threadpool with the `--jobs` pool, right?

Reply:
Plumbed this down into `buildOne`